tests/slow/src/test/scala/tests/sbt/SbtServerSuite.scala

```diff
@@ -33,23 +32,24 @@ class SbtServerSuite
 
   test("too-old") {
     cleanWorkspace()
+    writeLayout(
+      s"""|/project/build.properties
+          |sbt.version=$preBspVersion
+          |/build.sbt
+          |${SbtBuildLayout.commonSbtSettings}
+          |scalaVersion := "${V.scala212}"
+          |""".stripMargin
+    )
     for {
-      _ <- server.initialize(
-        s"""|/project/build.properties
-            |sbt.version=$preBspVersion
-            |/build.sbt
-            |scalaVersion := "${V.scala212}"
-            |""".stripMargin
-      )
+      _ <- server.initialize()
+      _ <- server.initialized()
       _ = assertNoDiff(
         client.workspaceMessageRequests,
         List(
-          importBuildMessage,
-          progressMessage
+          importBuildMessage
         ).mkString("\n")
       )
       _ = client.messageRequests.clear()
-      _ = assertStatus(_.isInstalled)
```

Comment:
Why remove this check?

Reply:
All tests used to run `bloopInstall` but that takes too long and it is not always useful. Here we want to test sbt server as the build server so we don't need to run `bloopInstall`, it is already tested in `SbtBloopLspSuite`.